api.planx.uk/saveAndReturn/validateSession.ts

```diff
@@ -96,13 +96,6 @@ export async function validateSession(
     const { reconciledSessionData, alteredSectionIds } =
       await reconcileSessionData({ sessionData, alteredNodes });
 
-    // store reconciled session data
-    await updateLowcalSessionData({
-      sessionId,
-      sessionData: reconciledSessionData,
-      email,
-    });
```

Comment:
I think session validation should be idempotent - it shouldn't change state and it should return the same response if called multiple times for the same account. 

If a user refreshes the resume page before continuing, they should see the same validation message and their session state should only be updated after they continue answering questions.

Reply:
This makese a lot of sense :+1: